docs/gen-datastructure.py

```diff
@@ -0,0 +1,32 @@
+# Tool for building the datastructure.rst file.
```

Comment:
Hopefully we can find a way round using this script, but for now i have not found a way.